sdk/servicebus/Azure.Messaging.ServiceBus/tests/Transactions/TransactionLiveTests.cs

```diff
@@ -479,27 +469,78 @@ public async Task TransactionGroupReceivesFirst(bool partitioned, bool enableSes
             Assert.IsNull(receivedMessage);
         }
 
+        // TODO - possible service bug involving subscription send via
```

Comment:
Can we uncomment and add an ignore instead?   That way, there's a reminder that this exists when tests are run....